web/src/services/prodPerfect.js

```diff
@@ -1,29 +1,66 @@
-!function(name,path,ctx){
-  var latest,prev=name!=='Keen'&&window.Keen?window.Keen:false;ctx[name]=ctx[name]||{ready:function(fn){var h=document.getElementsByTagName('head')[0],s=document.createElement('script'),w=window,loaded;s.onload=s.onreadystatechange=function(){if((s.readyState&&!(/^c|loade/.test(s.readyState)))||loaded){return}s.onload=s.onreadystatechange=null;loaded=1;latest=w.Keen;if(prev){w.Keen=prev}else{try{delete w.Keen}catch(e){w.Keen=void 0}}ctx[name]=latest;ctx[name].ready(fn)};s.async=1;s.src=path;h.parentNode.insertBefore(s,h)}}
-}('ProdPerfectKeen','https://replicated-ship-clusters.trackinglibrary.prodperfect.com/keen-tracking.min.js',this);
+// TODO: fix an enable eslint on this file
+/* eslint-disable */
```

Comment:
This was the only whole file I disabled eslint on. I don't know what this is but it doesn't look like a normal js file for this project.